openslides_backend/action/actions/meeting/import_.py

```diff
@@ -159,15 +138,12 @@ def update_meeting_users_and_mediafiles(self, instance: Dict[str, Any]) -> None:
         # set imported_at
         json_data["meeting"][0]["imported_at"] = round(time.time())
 
-        # delete blob from mediafiles
-        for entry in json_data.get("mediafile", []):
-            if "blob" in entry:
-                del entry["blob"]
-
     def create_replace_map(self, json_data: Dict[str, Any]) -> None:
         replace_map: Dict[str, Dict[int, int]] = {}
         for collection in json_data:
             replace_map[collection] = {}
```

Comment:
you could use a defaultdict here and remove this line

Reply:
Which variable should be a defaultdict?

Reply:
`replace_map`

Reply:
Ah, okay, you mean the `replace_map`.

Reply:
Done.